src/reports.js

```diff
@@ -11,6 +11,12 @@ import * as utils from './utils'
 
 config.reports = config.get('reports')
 
+let utcOffset = 0
```

Comment:
This appears to be an integer here when it is a String from the config. What does the moment package require?

Reply:
Maybe keep it consistent